### PR TITLE
Remove code moved to manageiq-smartstate repo

### PR DESCRIPTION
**Pull request by chessbyte**
See https://github.com/ManageIQ/manageiq-smartstate/blob/master/Rakefile

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -7,16 +7,6 @@ RSpec::Core::RakeTask.new(:spec) do |t|
...
-
-namespace :test do
-  Rake::TestTask.new(:extract)  { |t| t.test_files = ['test/ts_extract.rb'] }
-  Rake::TestTask.new(:metadata) { |t| t.test_files = ['test/ts_metadata.rb'] }
-  Rake::TestTask.new(:miq_disk) { |t| t.test_files = ['test/ts_mdfs.rb'] }
-end
-
 task :default do
   Rake::Task["spec"].invoke
   Rake::Task["test"].invoke
```
Comment (bdunne) [inline Rakefile]: I think this should go too, right?

Comment (jrafanie) [inline Rakefile]: Yes!  Good point @bdunne

Comment (chessbyte) [inline Rakefile]: No - there are still XML tests in `test` dir.

Comment (jrafanie) [inline Rakefile]: Oh, you didn't extract those yet 😊

**Pull request merged**